project/MimaFilters.scala

```diff
@@ -2,6 +2,16 @@ import com.typesafe.tools.mima.core.ProblemFilters.exclude
 import com.typesafe.tools.mima.core._
 
 object MimaFilters {
+  lazy val changesFor_3_0_0__RC5: Seq[ProblemFilter] = Seq(
+    // Incompatible signatures should not cause linking problems.
```

Comment:
Should be fine but we might want to repeat this test: https://github.com/typelevel/cats-effect/pull/609#pullrequestreview-280142297

Reply:
Are we sure they don't cause linking problems?

What happened there?

Reply:
Oh, it's due to the changed `Parallel` from Cats or something?

Reply:
If so, then there's nothing we can do there.